client/stylesheets/components/_generic.scss

```diff
@@ -14,7 +14,7 @@
 
 .full.screen {
   width: 100vw;
-  height: 80vh;
+  height: 100%;
```

Comment:
I mean I agree that the min-height should be 88vh. but why shouldn't the 100% be there?